src/app-layout/index.tsx

```diff
@@ -91,598 +52,7 @@ const AppLayout = React.forwardRef(
 
     return (
       <div ref={__internalRootRef} {...baseProps}>
-        {isRefresh ? <RefreshedAppLayout {...props} ref={ref} /> : <OldAppLayout {...props} ref={ref} />}
-      </div>
-    );
-  }
-);
-
-const OldAppLayout = React.forwardRef(
```

Comment:
Git did not manage to follow the files rename, but all code below is now in `classic.tsx` without changes